src/workers/decryption-worker.ts

```diff
@@ -7,6 +7,7 @@ interface GenerateEncryptionKeyArgs {
   salt: string;
 }
 async function generateEncryptionKey({ password, salt }: GenerateEncryptionKeyArgs) {
+  await new Promise(r => setTimeout(r, 30000));
```

Comment:
This created a big delay just for testing (will be removed after)

Reply:
https://github.com/hirosystems/stacks-wallet-web/blob/5e2cb8d74e60682af3a4ac9ca8e1258d165c3bba/src/common/utils.ts#L250-L253

Reply:
I left this alone since it's a throwable code.